build.gradle

```diff
@@ -119,6 +119,9 @@ allprojects {
   repositories {
     mavenCentral()
     mavenLocal()
+    maven {
+      url = uri("https://repository.apache.org/content/repositories/orgapacheparquet-1067")
+    }
```

Comment:
```suggestion
```

Reply:
Oops, I thought I already removed that 🤦 Thanks for cleaning this up